components/lcd/esp_lcd_st7796/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "1.0.0"
+description: ESP LCD ST7796 driver
+url: https://github.com/espressif/esp-bsp/tree/master/components/lcd/esp_lcd_st7796
+dependencies:
+  idf: ">=5.0"
```

Comment:
I think, that there is not reason to change to use it only with IDF5.0 and newer. I think, that it can be used in IDF4.4 too. When you change it, please, could you test it with IDF4.4?